build/meson/programs/meson.build

```diff
@@ -18,6 +18,7 @@ zstd_programs_sources = [join_paths(zstd_rootdir, 'programs/zstdcli.c'),
   join_paths(zstd_rootdir, 'programs/benchfn.c'),
   join_paths(zstd_rootdir, 'programs/benchzstd.c'),
   join_paths(zstd_rootdir, 'programs/datagen.c'),
+  join_paths(zstd_rootdir, 'programs/lorem.c'),
```

Comment:
> blindfix meson recipe
> 
> note: absence of GLOB capability within meson makes its maintenance more painful.

Very late but always feel free to ping me for this stuff. :)

Anyways, the Makefile *could* do globbing but doesn't, and cmake officially recommends that GLOB is a bad idea and should be avoided.

Alternative possibility is to have a text file called `sources.list` containing a single source filename on each line. Each build system could process it e.g. meson can do `zstd_programs_sources = fs.read('programs/sources.list').split('\n')`. Not sure anything can help the Visual Studio xml files. :(